extensions/wikia/AdMixExperiment/scripts/AdMixExperiment.js

```diff
@@ -288,7 +290,11 @@ require(['ext.wikia.adEngine.adContext', 'wikia.abTest', 'wikia.throttle'], func
 
 			// There is no recirculation right rail module on non-en wikis
 			if (window.wgContentLanguage === 'en') {
-				$recircWrapper.one('premiumRecirculationRail.ready', onRecircReady);
+				if ($recircWrapper.find('.premium-recirculation-rail').exists()) {
+					$recircWrapper.one('premiumRecirculationRail.ready', onRecircReady);
+				} else {
+					onRecircReady();
+				}
 			} else {
 				// TODO check which experiment needs it
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)